src/main/java/org/spongepowered/api/entity/player/Player.java

```diff
@@ -138,6 +137,38 @@
     void setGameMode(GameMode gameMode);
 
     /**
+     * Gets the location in dimension 0 of this player's bed spawn.
+     *
+     * @return The location of this player's bed spawn, or
+     * {@link Optional#absent} if it is not set
+     */
+    Optional<Vector3i> getBedPosition();
+
+    /**
+     * Sets the position in dimension 0 of this player's bed spawn. Passing
+     * {@link com.google.common.base.Optional#absent()} will clear it.
+     *
+     * @param location The new location of this player's bed spawn
+     */
+    void setBedPosition(Optional<Vector3i> location);
+
+    /**
+     * Returns whether this player is invisible to another given player.
+     *
+     * @param otherPlayer The player to toggle visibility of this player for
+     * @return Whether this player is invisible to another given player
+     */
+    boolean isInvisible(Player otherPlayer);
```

Comment:
You're adding several methods to `Human` but you keep this one in `Player`? Why is that? Doesn't this also interfere with [`Living#isInvisible()`](https://github.com/SpongePowered/SpongeAPI/blob/c9efe362b74acd456b3a1909285b0f3ceca29657/src/main/java/org/spongepowered/api/entity/living/Living.java#L282)?

Are you intending these methods to be used as 

``` java
if (!myPlayer.isInvisible(myOtherPlayer)) {
  myServer.sendEntityPacket(myOtherPlayer, new EntityPacket(myPlayer));
}
```

To me, this almost sounds like something VanishNoPacket would seek to achieve.